core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -48,6 +48,26 @@
    Map<Address, Response> invokeRemotely(Map<Address, ReplicableCommand> rpcs, RpcOptions options);
 
    /**
+    * Asynchronously sends the {@link ReplicableCommand} to the destination using the specified {@link DeliverOrder}.
+    *
+    * @param destination  the destination's {@link Address}.
+    * @param command      the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    */
+   void sendTo(Address destination, ReplicableCommand command, DeliverOrder deliverOrder);
```

Comment:
I was asking because I'm not convinced the replication time is relevant with the master algorithm either. (We need a name for it, maybe "sync forwarding"?) 

E.g. the RPCs from the primary to the backups are double-counted, because they're recorded explicitly on the primary and they're also recorded as part of the RPC from the originator to the primary. So I'm thinking about removing the RpcManagerImpl statistics altogether WDYT?